app/policies/comment_policy.rb

```diff
@@ -0,0 +1,17 @@
+# Pundit class to control authorization for comment actions.
+class CommentPolicy < ApplicationPolicy
+  # only signed up users can comment a book
+  def create?
+    user.present?
+  end
+
+  # only signed up users can update a comment
+  def update?
+    user.present?
```

Comment:
@valentina91p The reason I wrote this policy like that is because in the way I had previously written my controller signed in users only had access to their own comments when it came to the delete and update actions.
`comment = current_user.comments.find(params[:id]) `